addons/godot-firebase/auth/auth.gd

```diff
@@ -290,16 +291,17 @@ func get_auth_with_redirect(provider: AuthProvider) -> void:
 	else:
 		set_local_provider(provider)
 		OS.shell_open(url_endpoint)
-		print(url_endpoint)
+		#print(url_endpoint)
 
 
 # Login with Google oAuth2.
 # A token is automatically obtained using an authorization code using @get_google_auth()
 # @provider_id and @request_uri can be changed
 func login_with_oauth(_token: String, provider: AuthProvider) -> void:
 	if _token:
+		is_oauth_login = true
 		var token : String = _token.uri_decode()
-		print(token)
+		#print(token)
```

Comment:
Can you just remove the commented out lines altogether? I've been meaning to do that myself, this is a fine place to do it.

Reply:
Now it should be ok